src/com/github/arseniydubrovski/dev7/CheckData.java

```diff
@@ -0,0 +1,30 @@
+package com.github.arseniydubrovski.dev7;
+
+/**
+ * cheack data is valid or not
+ */
+public class CheckData {
+
+  /**
+   * Reply our data is valid or not
+   *
+   * @param inputData day/month/year and hours/minutes/seconds from command line
```

Comment:
what if day/month/year and hours/minutes/seconds were read from file and passed to this method? Based on your comment it must accept only values which came from command line but it actually accept any string from any sources